migrations/versions/0454_add_template_category.py

```diff
@@ -0,0 +1,76 @@
+"""
+
+Revision ID: 0454_add_template_categories
+Revises: 0453_add_callback_failure_email
+Create Date: 2024-06-11 13:32:00
+"""
+
+from datetime import datetime
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+revision = "0454_add_template_categories"
+down_revision = "0452_set_pgaudit_config"
+
+
+def upgrade():
+    op.create_table(
+        "template_categories",
+        sa.Column("id", postgresql.UUID(as_uuid=True), primary_key=True, nullable=False),
+        sa.Column("name_en", sa.String(length=255), nullable=False),
+        sa.Column("name_fr", sa.String(length=255), nullable=False),
+        sa.Column("description_en", sa.String(length=255), nullable=True),
+        sa.Column("description_fr", sa.String(length=255), nullable=True),
+        sa.Column("sms_process_type", sa.String(length=255), nullable=False),
+        sa.Column("email_process_type", sa.String(length=255), nullable=False),
+        sa.Column("hidden", sa.Boolean(), nullable=False),
+        sa.UniqueConstraint("name_en"),
+        sa.UniqueConstraint("name_fr")
+    )
+
+    op.add_column(
+        "templates",
+        sa.Column("template_category_id", postgresql.UUID(as_uuid=True), nullable=True)
+    )
+    op.create_index(
+        op.f("ix_template_category_id"),
+        "templates",
+        ["template_category_id"],
+        unique=False,
+    )
+    op.create_index(
+        op.f("ix_template_categories_name_en"),
+        "template_categories",
+        ["name_en"],
+        unique=False,
+    )
+    op.create_index(
+        op.f("ix_template_categories_name_fr"),
+        "template_categories",
+        ["name_fr"],
+        unique=False,
+    )
+    op.create_foreign_key(
+        "fk_template_template_categories",
+        "templates",
+        "template_categories",
+        ["template_category_id"],
+        ["id"]
+    )
+
+    # Insert the generic Low priority (bulk) category
```

Comment:
lets add this! but not just got bulk, we want to do "default high", med, low. Also lets store the IDs in the config as they are magic values (or if not confg, then defs on top of this file as globals)